index.html

```diff
@@ -13,51 +13,48 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,600;1,700;1,800&family=Roboto:wght@300;400;500&family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/b2291adafa.js" crossorigin="anonymous"></script>
     
-    <title>Document</title>
+    <title>Esther's Portfolio</title>
 </head>
 <body>
- <div id="container">
-        
-    <section id="wrapper">
-            <!--Header Section -->
-        <section id="header">
-            <nav class="nav">
-                <h3 class="logo">My Logo</h3>
-                <div class="menu-icon">
-                    <i class="uil uil-bars"></i>
-                </div>
-            </nav>
-        </section>
-        
-        <!-- Header Desktop -->
-        <section id="header-desktop">
-            <nav class="nav">
-                <h3 class="logo">My Logo</h3>
-                <ul class="menu-icon">
-                    <li><a href="">Portfolio</a></li>
-                    <li><a href="">About</a></li>
-                    <li><a href="">Contact</a></li>
+    <!--Header Section -->
+    <header>
+        <div class="container">     
+            <div class="mobile-logo">
+                <h3 id="logo">My Logo</h3>
+            </div>
+            <div class="mobile-menu">
+                <i class="uil uil-bars navtoggle"></i>
+            </div> 
```

Comment:
You did well implementing the changes requested by the previous reviewer. However, the hamburger button is still different from the one in the Figma design.
Please change the font size in line 73 in your css file from `font-size: x-large`  To `font-size: xx-large` and it will fix the issue.